app/services/content_item_publisher.rb

```diff
@@ -1,13 +1,13 @@
 class ContentItemPublisher
-  def publish(flow_presenters)
+  def sync(flow_presenters)
     flow_presenters.each do |smart_answer|
       start_page_content_item = StartPageContentItem.new(smart_answer)
       Services.publishing_api.put_content(start_page_content_item.content_id, start_page_content_item.payload)
-      Services.publishing_api.publish(start_page_content_item.content_id)
+      Services.publishing_api.publish(start_page_content_item.content_id) if smart_answer.publish?
```

Comment:
Is this okay when a flow goes from published back to draft? Wasn't sure how Publishing API deals with that

Reply:
No it doesn't handle that situation well at all unfortunately. Publishing API would have a new draft of the items and the previous one would be published so you'd be served the Smart Answer still.

If you go from published back to draft you'd have to follow the retiring steps first, then this would be fine.

Reply:
Ah righto. The naming `sync` maybe a little confusing against our publishing model. Probably something we can address in a future PR along with some more docs about the draft vs published states.

Reply:
Yeah - I'm not very happy about the naming. I felt that `#publish` was wrong but `#sync` isn't as sophisticated as you may expect.